tests/test_routing.py

```diff
@@ -223,6 +236,10 @@ def test_basic_building():
     assert adapter.build("foo", {}) == "/foo"
     assert adapter.build("foo", {}, force_external=True) == "//example.org/foo"
 
+    adapter = map.bind("example.org", url_scheme="ws")
+    assert adapter.build("ws", {}) == "/ws"
+    assert adapter.build("foo", {}, force_external=True) == "ws://example.org/foo"
```

Comment:
Will `force_external` when the request is `https` create a `wss` URL?

Reply:
And should websocket routes always build external urls since they require the `ws://` scheme?

Reply:
Agree, changed.